website/markdown/posts/2020-04-28-why-tuist/post.mdx

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: Why Tuist?
+categories:
+  [
+    tuist,
+    xcode project generation,
+    xcode,
+    swift,
+    scale,
+    ios,
+    projects,
+    scalability,
+  ]
+excerpt: In this blog post I share my thoughts on why I think Tuist is a good choice to scale up Xcode projects, and guide the reader through what I believe are key features to make that easy.
+author: pepibumur
+---
+
+If you are considering the adoption of Tuist,
+you might wonder _why_ a project like yours should make such a move.
+The idea of adding a new tool to your toolbox might sound intimidating ―
+it certainly is.
+But believe me,
+this is one that you won't regret about.
+Despite having done our best to convey the idea behind Tuist,
+and why it's an important piece when scaling up projects,
+I feel we lacked a good summary
+
+## Make your projects consistent and conventional
+
+After bootstrapping new projects and targets in Xcode,
+it's up to the developers to ensure that projects are consistently configured and structured.
+Some developers might see that as something great,
+but that comes with some downsides:
+
+- It adds **indirection** for developers when jumping between projects: _How is this project structured compared to this other?_ _Why is this target linking using build settings flags while the other one is using build phases?_
+- Xcode's **indexing and build system** might not work as effectively because it needs to resolve configuration nuances across all the projects.
+  Moreover, those nuances might be conflicting and result in compilation errors.
+
+You might have seen teams solving this problem at the build settings level by extracting them into reusable [Xcode Build Configuration Files](https://nshipster.com/xcconfig/).
+Still, we should not disregard that settings are just a piece of the cake ―
+There are also build phases, targets, or schemes, for which Xcode doesn't provide any way to reuse them.
+Other teams resort to scripts that run some checks on the projects,
+but that results in a poor experience for developers because those checks are not built into their workflows.
+
+Tuist solves this by providing [project description helpers](/docs/usage/helpers/).
+All you need to do is to define what types of projects are supported,
+and codify them into functions that return templated projects:
+
+```swift
+imporrt ProjectDescription
```

Comment:
There's a r too many in the import

Reply:
`imporrt` is the new way of importing things :trollface: